config/application.properties

```diff
@@ -38,4 +38,8 @@ quarkus.application.name=strimzi-kafka-bridge
 #using OTEL_SERVICE_NAME as usual or a preformatted service name based on the bridge.id
 quarkus.opentelemetry.tracer.resource-attributes=service.name=${OTEL_SERVICE_NAME:strimzi-kafka-bridge-${bridge.id}}
 #if the OTEL_EXPORTER_OTLP_ENDPOINT, it's using http://localhost:4317 by default
-quarkus.opentelemetry.tracer.exporter.otlp.endpoint=${OTEL_EXPORTER_OTLP_ENDPOINT}
\ No newline at end of file
+quarkus.opentelemetry.tracer.exporter.otlp.endpoint=${OTEL_EXPORTER_OTLP_ENDPOINT}
+
+#Inactive consumer timeout configuration
+#This property is added so that you can configure the http.timeoutSeconds the usual way.
```

Comment:
Tbh I think we should explain differently. Not related to the timeout but saying that it's needed to allow scheduling a periodic task programmatically.